department-of-veterans-affairs/caseflow
# Add Modal state bugs.

Resolves #8367 

### Description
Reset the regional office dropdown when reopening add modal. Also fixed an issue with the index used to add new hearing day to hearingSchedule object structure.

### Acceptance Criteria

### Testing Plan
1. Go to View Hearings
2. Add a video hearing and confirm
3. Verify the new hearing day in the View Hearings page
4. Add a central office hearing
5. Verify that all dropdowns are reset when modal is opened
6. Fill out form and confirm
7. Verify the central office hearing is added to View Hearings and that it has no regional office value.
